builtin/providers/aws/resource_aws_api_gateway_resource.go

```diff
@@ -0,0 +1,143 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/apigateway"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsApiGatewayResource() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsApiGatewayResourceCreate,
+		Read:   resourceAwsApiGatewayResourceRead,
+		Update: resourceAwsApiGatewayResourceUpdate,
+		Delete: resourceAwsApiGatewayResourceDelete,
+
+		Schema: map[string]*schema.Schema{
+			"api_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"parent_resource_id": &schema.Schema{
```

Comment:
Shall we change this field name to match the API? i.e. `parent_resource_id` -> `parent_id`?